CountItems/masterProcess.js

```diff
@@ -18,6 +22,14 @@ const loggerConfig = {
 werelogs.configure(loggerConfig);
 const log = new werelogs.Logger('S3Utils::CountItems::Master');
 
+const prometheusPollingFrequency = process.env.PROMETHEUS_POLLING_FREQUENCY && !Number.isNaN(process.env.PROMETHEUS_POLLING_FREQUENCY)
+    ? Number.parseInt(process.env.PROMETHEUS_POLLING_FREQUENCY, 10)
+    : 30;
+
+const promtheusPollingThreshold = process.env.PROMETHEUS_POLLING_THRESHOLD && !Number.isNaN(process.env.PROMETHEUS_POLLING_THRESHOLD)
```

Comment:
can't speak of a "threshold" here : polling is an action, not a measurable...

Reply:
rename to `prometheusPollingAttempts` ( any better naming idea is welcome ... ) => https://github.com/scality/s3utils/pull/312/commits/7a546979810051418f9af9bc3ed2567a0bc69eb4#diff-46cbf734fd7b9022c7bbf597a8a5deb96829b581ca817a861bbc3bf1e86274a9R31-R32